jobs_detector/helpers.py

```diff
@@ -0,0 +1,63 @@
+import re
+from collections import defaultdict
+
+def _keyword_search(comment, keyword):
+    """
+    Return True if the keyword is found in the search and add to keywords_dict
+    and comments_dict dictonaries.
+    
+    keywords_dict = {Python:[12345,09876,45677], 
+                     Remote:[12345,98752,15973]}
+                    
+    comments_dict = {12345:'This is the comment test of the post'}
+                    {98765:'This is another comment test of the post'}                
+    """
+    if re.search(re.compile(keyword, re.IGNORECASE), str(comment)):
+        parent = comment.find_parent('tr', {'class':'athing'})
+        get_id = parent.find('a', {"id":re.compile('up_.*')})['id'].split('_')[1]
```

Comment:
There's a lot going on on this line, maybe the logic could be split up over several lines?